### PR TITLE
Pull Task6 and 7

### PR DESCRIPTION
**Pull request by AndriiLata**

### CODE REVIEW

File: Assignment4/AndriiLata - 230762/task06.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the email address, fullName, given and family names**"""
+
+g.add((ns.JaneSmith, ns.email, Literal("janesmith@gmail.com")))
+g.add((ns.JaneSmith, ns.fullName, Literal("Jane Smith")))
+g.add((ns.Jane_mith, ns.Given, Literal("Jane")))
+g.add((ns.JaneSmith, ns.familyName, Literal("Smith")))
```
Comment (anaigmo) [inline Assignment4/AndriiLata - 230762/task06.py:56]: use the properties from VCARD (you can check them in the already existing graph)

File: Assignment4/AndriiLata - 230762/task07.py
Comment (anaigmo) [inline Assignment4/AndriiLata - 230762/task07.py:1]: missing rdflib alternatives for tasks 7.1-3

File: Assignment4/AndriiLata - 230762/task07.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+"""**TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**"""
+
+q1 = f"""SELECT ?subclass
+        WHERE {{ ?subclass rdfs:subClassOf ns:LivingThing. }} """
+for r in g.query(q1):
+  print(r.subclass)
```
Comment (anaigmo) [inline Assignment4/AndriiLata - 230762/task07.py:29]: and subclasses of subclasses?

File: Assignment4/AndriiLata - 230762/task07.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+"""**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**
+
+"""
+
+q2 = f"""SELECT ?individuals
+        WHERE {{ ?individuals a ns:Person . }}"""
+
```
Comment (anaigmo) [inline Assignment4/AndriiLata - 230762/task07.py:37]: not taking into account subclasses of :Person! (and instances of their subclases, and so on)

File: Assignment4/AndriiLata - 230762/task06.py (hunk 1 above)
Comment (AndriiLata) [reply Assignment4/AndriiLata - 230762/task06.py:56]: Here are my changes, I dont know why, but I cannot pull it again. I tried everything it doesnt work. If you really want me to do a pull request with only those details changed, pls explain how.

g.add((ns.JaneSmith, ns.email, Literal("janesmith@gmail.com")))
g.add((ns.JaneSmith, ns.FN, Literal("Jane Smith")))
g.add((ns.Jane_mith, ns.Given, Literal("Jane")))
g.add((ns.JaneSmith, ns.Family, Literal("Smith")))

File: Assignment4/AndriiLata - 230762/task07.py
Comment (AndriiLata) [reply Assignment4/AndriiLata - 230762/task07.py:1]: Here are my changes, I dont know why, but I cannot pull it again. I tried everything it doesnt work. If you really want me to do a pull request with only those details changed, pls explain how.

## TASK 7.1: 
""#RDFLIB
ns = Namespace("http://somewhere#")
lts = set()
for subj, obj in g.subject_objects(predicate=RDFS.subClassOf):
    if obj == ns.LivingThing:
        lts.add(subj)
print('RDFLIB:')
for subclass in lts:
    print(subclass)

## TASK 7.2: 
""#rdflib
p_i = set()
for subj, _, _ in g.triples((None, RDF.type, ns.Person)):
    p_i.add(subj)

""#results
print('RDFLIB:')
for person_individual in p_i:
    print(person_individual)

## TASK 7.3: 
""#RDFLIB
individuals = set()
for subj, _, obj in g.triples((None, RDF.type, ns.Person)):
    individuals.add(subj)
for subj, _, obj in g.triples((None, RDF.type, ns.Animal)):
    individuals.add(subj)

File: Assignment4/AndriiLata - 230762/task07.py (hunk 2 above)
Comment (AndriiLata) [reply Assignment4/AndriiLata - 230762/task07.py:29]: Here are my changes, I dont know why, but I cannot pull it again. I tried everything it doesnt work. If you really want me to do a pull request with only those details changed, pls explain how.

q1 = f"""SELECT ?subclass
        WHERE {{ ?subclass rdfs:subClassOf* ns:LivingThing. }} """

**Comment by anaigmo** [comment]
Entregado con correcciones, no se aceptan más PR

**Pull request merged**